src/components/orders/crud/OrderDetail.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from 'react'
+import { Container, Row, Col } from 'react-bootstrap/'
+import DetailElement from '../../common/DetailElement'
+import OrdersApiService from '../../../utils/api/ordersApiService'
+import './OrderDetail.scss'
+
+export default function OrderDetail(props) {
+	const [order, setOrder] = useState(null)
+	const [customer, setCustomer] = useState(null)
+	const [productsList, setProductsList] = useState([])
+
+	useEffect(() => {
+		const fetchData = async () => {
+			const fetchedOrder = await OrdersApiService.getOrderById(props.match.params.id)
+
+			setOrder(fetchedOrder.data)
+			setCustomer(fetchedOrder.data.customer)
+			setProductsList(fetchedOrder.data.products)
+		}
+		fetchData()
+	}, [])
+
+	return (
+		<div className="order-detail">
```

Comment:
I think you can remove this div, add this className to a Container below and move `{order && customer ? (` to be inside of the container rather than outside

Reply:
done